tests/test_sourmash.py

```diff
@@ -1940,6 +1940,84 @@ def test_search_traverse_incompatible(c):
 # explanation: you cannot downsample a scaled SBT to match a scaled
 # signature, so make sure that when you try such a search, it fails!
 # (you *can* downsample a signature to match an SBT.)
+
```

Comment:
the comment above this belongs with the function `test_search_metagenome_downsample`.

Reply:
okay